README.md

```diff
@@ -1 +1,66 @@
-# Conference-Project
\ No newline at end of file
+# Microverse-Project-1
+
+## Description
+
+This website is my personal portoflio. I have developed number of websites and the projects and my achievents are available here
+
+## Built With
+
+- HTML
+
+## Live Demo
+
+[Live Demo Link](https://aseadd.github.io)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Project Setup
+
+To get a local copy and running follow this steps
+prerequisites
+you need to have code editor
+Make sure NodeJs and NPM are installed
+
+### Setup
+
+1.  Clone the repo into your local machine
+    `git clone https://github.com/Aseadd/Microverse-Project-1
+2.  open terminal and cd in the project
+    `cd Microverse-Project-1
+3.  Open the index html in your browser
+    `open index.html
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Aseadd)
+- Twitter: [@twitterhandle](https://twitter.com/adaTsega)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/addistsega)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] it would be important to acknowledge[ Cindy Shin](https://www.behance.net/adagio07) tthe [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).